jsx/App/Stories/Story/lib/txt_sync.js

```diff
@@ -135,3 +149,45 @@ export function setupTextSync() {
     });
   
 }
+
+
+export function setupYoutube() {
+
+    const youtubeMedia = document.querySelectorAll("[is-youtube='true']")[0];
+    if (!youtubeMedia) {
+        return; 
+    }
+
+    const youtubeID = youtubeMedia.getAttribute('youtube-id');
+
+    // Initialize YouTube player: 
+    window.YT.ready(function() {
+        player = new window.YT.Player("video", {
+            height: "270",
+            width: "480",
+            videoId: youtubeID,
+            events: {
+                'onReady': onPlayerReady,
+                'onStateChange': onPlayerStateChange
+            }
+        });
+    });
+
+    function onPlayerReady(event) {
```

Comment:
These two functions have empty bodies. Do we need them? 

Usually, event listeners like this can be omitted when you don't care about the event. I can imagine a couple good reasons for keeping these (maybe you think future LingView developers will want to add function bodies, or maybe the YouTube API is badly behaved and throws an error if you don't define these), but it should be a deliberate choice.

Reply:
I originally thought these two listeners need to be present for the YouTube API, but that turns out to not be the case. I will delete these two functions.